CodeChat_Server/windows_installer.ps1

```diff
@@ -0,0 +1,205 @@
+
+        # Installs Python ``Out-Null`` option makes sure the program doesn't continue before the installation completes Using Start-Process over just using the cmd because it allows me to input a string, so we can easily change the target python version. 
+        Start-Process ".\python-$PyTarget-amd64.exe" -ArgumentList /quietly,PrependPath=1 -Wait
+
+        echo "Python Version $PyTarget Installed Successfully
+        "
+    }
+    else{
+        echo "Please install the required version of python: $pythonVersionReq or later
+        "
+        exit
+    }
+}
+else{
+    echo "Acceptable Version of Python found:"
+}
+
+# Pip Installation
+# ----------------
+# Make sure pip, the Python installer, is up to date on Windows
+echo "Upgrading pip..."
+python -m pip install --upgrade pip
+echo "
+"
+
+# Going to the user dir
+cd $env:USERPROFILE
+
+# Creating codechat venv
+# ----------------------
+# We do this so that our installation does not mess with any other installations of python
+echo "Creating codechat venv..."
+$codechat = Get-Item codechat
+if([string]::IsNullOrEmpty($codechat)){
+    # Create a virtual enviroment named codechat
+    python -m venv codechat
+    echo "Virtual Enviroment Created Sucessfully
+    "
+}
+else{
+    echo "'codechat' virtual enviroment already found, skipping this step
+    "
+}
+
+echo "Activating Virutal enviroment"
+# Activate the virtual enviroment
+.\codechat\Scripts\activate
+echo "Virtual Enviroment Activated
+"
+
+# find CodeChat_Server.exe and tell user if just updating or installing
+# This could probably be changed, so that it doesn't differentiate seeing as both paths do that same thing.
+$CodeChat_Server = Get-Command $env:USERPROFILE\codechat\Scripts\CodeChat_Server
+if([string]::IsNullOrEmpty($CodeChat_Server)){
+    # Install the CodeChat Server
+    echo "installing CodeChat_Server"
+    python -m pip install --upgrade CodeChat_Server
+    echo "CodeChat_Server Sucessfully Installed
+    "
+}
+else{
+    # Update the CodeChat Server
+    echo "CodeChat_Server found, running update"
+    python -m pip install --upgrade CodeChat_Server
+    echo "CodeChat_Server sucessfully updated
+    "
+}
+
+# I decided to add an automatic download of the myst parser, because if you're working on codechat you will need it.
+$MystParser = python -m pip show myst-parser
+if([string]::IsNullOrEmpty($MystParser)){
+    echo "Did not find myst_parser installing..."
+    python -m pip install myst-parser
+}
+
+
+# `Set-Clipboard <https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-clipboard?view=powershell-7.2>`_: Copies path of CodeChat_Server to the clipboard for easy pasting and displays path in terminal
+$pathToCodeChat = Get-Command $env:USERPROFILE\codechat\Scripts\CodeChat_Server | Select-Object -ExpandProperty Definition
+Set-Clipboard $pathToCodeChat
+echo "Here is your path to CodeChat (Also copied to your clipboard): $pathToCodeChat"
+
+# Exit the Script
```

Comment:
Omit.

Reply:
to clarify: this currently copies the incorrect path (single backslashes) into the user's clipboard

We want it to:
A. copy correct path
B. Not copy path, just `echo` and tell the user to copy it and paste it into VSC
C. 'echo' path and provide link to install directions
D. don't do anything with the path; just exit

Reply:
The "Omit" referred only to the "press enter to exit" message.

For the above question: option A (unless that's difficult).

Reply:
Actually, instead of option A, let's ignore this problem. I'll add a commit which no longer needs double backslashes.

Reply:
removed the press enter to exit message